src/block_cache/block_chunkcache.c

```diff
@@ -1032,6 +1117,28 @@ __wt_chunkcache_teardown(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __wt_chunkcache_salvage --
+ *     Remove any knowledge of any extant chunk cache metadata. We can always rebuild the cache
+ *     later, so make no attempt at a "real" salvage.
+ */
+int
+__wt_chunkcache_salvage(WT_SESSION_IMPL *session)
+{
+    WT_DECL_RET;
+    const char *drop_cfg[] = {WT_CONFIG_BASE(session, WT_SESSION_drop), NULL};
+
+    /* Check that we're holding the schema lock (or take it) before doing a schema operation. */
+    if (FLD_ISSET(session->lock_flags, WT_SESSION_LOCKED_SCHEMA))
+        ret = __wt_schema_drop(session, WT_CC_METAFILE_URI, drop_cfg);
+    else
+        WT_WITH_SCHEMA_LOCK(session, ret = __wt_schema_drop(session, WT_CC_METAFILE_URI, drop_cfg));
```

Comment:
Shall we also get rid of the chunkcache file in this case?

Reply:
That's a fair question, but there's no need - we always start with an empty allocation bitmap, so there's never a need to drop the file to prevent reuse.

(Perhaps there are other cases where we should drop the chunk cache file - but that's out of scope for this ticket, which is for managing the metadata file specifically.)